tools/exposure-client/main.go

```diff
@@ -92,33 +104,48 @@ func main() {
 		Padding:             base64.RawStdEncoding.EncodeToString(padding),
 	}
 
-	prettyJSON, err := json.MarshalIndent(data, "", "  ")
+	body, err := json.MarshalIndent(data, "", "  ")
 	if err != nil {
-		log.Printf("Can't display JSON that was sent, error: %v", err)
-	} else {
-		log.Printf("SENDING: \n%v", string(prettyJSON))
+		return fmt.Errorf("failed to generate JSON: %w", err)
 	}
+	fmt.Printf("generated json: \n%s\n", body)
```

Comment:
Did you intend to leave this in?

Reply:
Yea - it's nice for debugging/replaying the request. I figured I'd leave it.